manual/query_builder/idempotence/README.md

```diff
@@ -99,7 +99,7 @@ Statement statement =
         .appendListElement("l", literal(1))
         .whereColumn("k").isEqualTo(bindMarker())
         .build();
-// UPDATE foo SET l+=[1] WHERE k=?
```

Comment:
this change confuses me, I think that the goal of the ticket is to support `+=` syntax, but here instead we are changing the syntax to `value= value+...`

Reply:
No the goal of the ticket is not to support `x += y`. This is the currently supported syntax, but it has the drawback of not being available for C* < 3.10. The goal is to use the old, alternative syntax `x = x + y`, which is more widely supported.